sdcm/cluster.py

```diff
@@ -1903,13 +1917,24 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         """.format(mgmt_user=mgmt_user, rsa_id_dst=rsa_id_dst, rsa_id_dst_pub=rsa_id_dst_pub))  # generate ssh public key from private key.
         self.remoter.run('sudo bash -cxe "%s"' % ssh_config_script)
 
+        self.config_scylla_manager_yaml(segments_per_repair=100)
+
         if self.is_docker():
             self.remoter.run('sudo supervisorctl restart scylla-manager')
             res = self.remoter.run('sudo supervisorctl status scylla-manager')
         else:
             self.remoter.run('sudo systemctl restart scylla-manager.service')
             res = self.remoter.run('sudo systemctl status scylla-manager.service')
 
+        if self.is_rhel_like():
+            configuring_new_manager_command = dedent("""
```

Comment:
what's `new` has todo with this parameter 
also add a TODO comment regarding debian/ubuntu like installations